wgkex/worker/mqtt.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+import paho.mqtt.client as mqtt
+import socket
+import time
+import re
+from wgkex.worker.netlink import (
+    find_stale_wireguard_clients,
+    link_handler,
+    WireGuardClient,
+    generate_lladdr,
+    generate_interface_names,
+)
+
+
+def connect(domains: str):
+    broker_address = "broker.ov.ffmuc.net"
+    client = mqtt.Client(socket.gethostname())
+    client.on_message = on_message
+    print("connecting to broker " + broker_address)
+    client.connect(broker_address)
+    for domain in domains:
+        print("Subscribing to topic", "wireguard/" + domain + "/+")
+        client.subscribe("wireguard/" + domain + "/+")
```

Comment:
If all workers subscribe tot the same generic path, how is the gateway ultimately selected?

Reply:
The node selects the gateway at the moment. There is no feedback channel from the Broker.